src/provider/handler/getPosts.js

```diff
@@ -25,7 +41,7 @@ export function getPosts(params) {
       let mediaPromises = [];
       result.entry.forEach(result=>{
```

Comment:
instead of allocating an array for this, you can also do something like
```
return Promise.all(result.entry.map(result => axios.get(...))
  .then(results => ...)
```

what you can extract is the mapResultToPromise function, for readability, and then you can can simply invoke `Promise.all(result.entry.map(mapResultToPromise))`